regional_mom6/regional_mom6.py

```diff
@@ -797,7 +797,16 @@ def ocean_forcing(
         vel_out = vel_out.interp({"zl": self.vgrid.zl.values})
 
         print("Saving outputs... ", end="")
-        vel_out.fillna(0).drop("time").to_netcdf(
+
+        ## Remove time IF it exists. Users may already have done so for us
+        if "time" in vel_out.dims:
+            vel_out = vel_out.isel(time=0).drop("time")
+        if "time" in vel_out.dims:
+            tracers_out = tracers_out.isel(time=0).drop("time")
+        if "time" in vel_out.dims:
+            eta_out = eta_out.isel(time=0).drop("time")
```

Comment:
```suggestion
        for out in [vel_out, tracers_out, eta_out]:
            if "time" in out.dims:
                out = out.isel(time=0).drop("time")
```

Reply:
This won't work, you will only be overwriting the `out` variable, not the "actual" variable. This is just re-implementing `.squeeze("time")`, probably just use that, i.e.

```python
vel_out.squeeze("time").fillna(0)...
```

etc.

Reply:
ok, then perhaps we can leave it as it was initially?
or just fix the typos (I believe there were few typos) 

```suggestion
        if "time" in vel_out.dims:
            vel_out = vel_out.isel(time=0).drop("time")
        if "time" in tracers_out.dims:
            tracers_out = tracers_out.isel(time=0).drop("time")
        if "time" in eta_out.dims:
            eta_out = eta_out.isel(time=0).drop("time")
```

Reply:
otherwise, if indeed the if statement should be the same for all, we can combine under one if:

```suggestion
        if "time" in vel_out.dims:
            vel_out = vel_out.isel(time=0).drop("time")
            tracers_out = tracers_out.isel(time=0).drop("time")
            eta_out = eta_out.isel(time=0).drop("time")
```

Reply:
Probably the second suggestion -- not sure we can assume if one file has time, they all do? I will have a look at how to streamline all of this, not particularly clean in any configuration.

Reply:
The reason for this is that it's not guaranteed that the user has already removed the 'time' dim from the initial conditions. EG, GLORYS dataset leaves time in, but when I made my dummy data there was no time dim so original implementation threw an error. I figured that if it's an "Initial condition" it would be intuitive that the user might delete the time dim so wanted to handle this